src/View/Activity/activity.html.twig

```diff
@@ -9,12 +9,11 @@
 	<h1 class="text-center m-3">Les évènements</h1>
 	<div class="row justify-content-evenly text-light">
 		{% for event in events %}
-
 			<div class="card bg-success col-3 m-3 p-0" style="width: 18rem;">
 				{% if event.picture_link == null %}
-					<img class="activityImg rounded" src="/assets/images/imgPlaceHolder.jpg" alt="eventPicture" srcset="">
+					<img class="eventsImg rounded" src="/assets/images/imgPlaceHolder.jpg" alt="eventPicture" srcset="">
```

Comment:
tu pourrais le faire un une ligne avec qq chose comme
`<img class="eventsImg rounded" src="{{ event.picture_link is null ? '/assets/images/imgPlaceHolder.jpg' : event.picture_link  }}" alt="photo de l'événement">`